src/components/Theme/style.css

```diff
@@ -9,6 +9,15 @@
   flex-direction: column;
 }
 
+/* this will only apply styles to Edge */
+@supports (-ms-ime-align:auto) {
+  /* Prevent overflow on fullscreen on Edge */
+  .fullScreenStep {
```

Comment:
I get that it's not a problem on other browsers - but is there any reason we can't do this for all browsers?

Reply:
No reason. I've changed it :)